### ISSUE TITLE
Sample config_inc.php?

### ISSUE THREAD

**Issue by lightweight** [novatechweb/docker-mantisbt#1]
Many thanks for making your project available on github... Seems to be just what I've been looking for. Just wondering if you might be able to commit a, say, sample.config_inc.php file with some mock data just to see what format you've used and the full selection of Env variables you're setting?

**Comment by JosephLutz** [comment]
We did not wish to place the config file in the repository as it would be a file that is not used by any of the code. Depending upon your setup the settings could be drastically different. When I created the config file I used our old config as a starting point for what settings we needed. I then went through all the possible settings in the new version of mantis and decided weather it would be better for our setup to set them or not.

This is our current config file, with all values changed that could be specific to NovaTech.
[sample.config_inc.php.txt](https://github.com/novatechweb/docker-mantisbt/files/445050/sample.config_inc.php.txt)
[The values I changed have 'CHANGED_' and then the variable name as the new value from the file.]
